velox/functions/prestosql/tests/ArrayContainsTest.cpp

```diff
@@ -207,21 +200,12 @@ TEST_F(ArrayContainsTest, booleanWithNulls) {
       {false, false, false},
   });
 
-  auto testContains = [&](std::optional<bool> search,
-                          const std::vector<std::optional<bool>>& expected) {
-    auto result = evaluate<SimpleVector<bool>>(
-        "contains(c0, c1)",
-        makeRowVector({
-            arrayVector,
-            makeConstant(search, arrayVector->size()),
-        }));
-
-    assertEqualVectors(makeNullableFlatVector<bool>(expected), result);
-  };
-
-  testContains(true, {true, true, std::nullopt, false, true, false});
-  testContains(false, {true, false, true, false, true, true});
-  testContains(
+  testContains<bool>(
```

Comment:
curious why did you need to pass the type?
is it generating ambiguous calls?
if so shall we instead have 
testContains
testContainsCosntantKey

Reply:
Yes, it would generate ambiguous calls. Fixed it by using testContains and testContainsCosntantKey.